### PR TITLE
Fix/feedback 07 11 2024

### PR DESCRIPTION
**Pull request by Kishanackplus**

### CODE REVIEW

File: apps/admin/src/app/components/custom-breadcrumbs/linkItem.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { Link as RouterLink } from 'react-router-dom';
```
Comment (ajay-khandla) [inline apps/admin/src/app/components/custom-breadcrumbs/linkItem.tsx:1]: Use a small case file name

File: apps/admin/src/app/components/form/autocomplete.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  //       } else {
+  //         item = find(options, { [renderKey]: val });
+  //       }
+  //       onChange && onChange(e, item, 'blur');
+  //     }
+  //   },
+  //   [freeSolo, multiple, props, onChange, options, renderKey]
+  // );
+console.log(autocompleteValue);
+
```
Comment (ajay-khandla) [inline apps/admin/src/app/components/form/autocomplete.tsx:94]: Remove logs

File: apps/admin/src/app/sections/auth/verify-otp/Index-verify.tsx [hunk 3]
```diff
@@ -77,7 +77,7 @@ const IndexVerify = ({
 			{!isValidOtp ? (
 				<Container maxWidth="sm">
 					<ContentStyle>
-						<Typography variant="h1">
```
Comment (ckhandla94) [inline apps/admin/src/app/sections/auth/verify-otp/Index-verify.tsx]: file name should be in lower case

File: apps/admin/src/app/sections/role/add-edit-role-dialog.tsx [hunk 4]
```diff
@@ -15,38 +18,49 @@ export interface AddEditRoleDialogProps {
...
-  //     }
-  //   },
-  //   [],
-  // )
+    }).catch(() => {
+      // setUserRoles(null)
+    })
+  }, [])
 
+  console.log(roleValue);
```
Comment (ajay-khandla) [inline apps/admin/src/app/sections/role/add-edit-role-dialog.tsx:41]: Remove console log

File: apps/admin/src/app/sections/role/add-edit-role-dialog.tsx [hunk 5]
```diff
@@ -15,38 +18,49 @@ export interface AddEditRoleDialogProps {
...
+  useEffect(() => {
+    if (roleValue?.id) {
+      roleService.getOne(roleValue?.id, {
+        relations: ['permissions'],
+        select: {
+          permissions: { id: true }
+        }
+      }
+      ).then((resp) => {
+        console.log(resp);
```
Comment (ajay-khandla) [inline apps/admin/src/app/sections/role/add-edit-role-dialog.tsx:51]: Remove logs

File: apps/api/src/app/module/user/user.service.ts [hunk 6]
```diff
@@ -24,18 +25,24 @@ export class UserService extends CrudService<User> {
     super(userRepository);
   }
 
-  async beforeSave(entity: DeepPartial<User>, req): Promise<User> {
-    // if ((entity as any).password) {
-    //   entity.passwordHash = await hashPassword((entity as any).password.trim());
-    // }
+  async beforeSave(entity: DeepPartial<any>, req): Promise<User> {   
+    if (has(req, 'password') && req?.password ) {
+      console.log(65655656565);
```
Comment (ajay-khandla) [inline apps/api/src/app/module/user/user.service.ts:30]: Remove console log

File: libs/react-core/tsconfig.lib.json [hunk 7]
```diff
@@ -20,5 +20,5 @@
     "src/**/*.spec.jsx",
     "src/**/*.test.jsx"
   ],
-  "include": ["src/**/*.js", "src/**/*.jsx", "src/**/*.ts", "src/**/*.tsx"]
+  "include": ["src/**/*.js", "src/**/*.jsx", "src/**/*.ts", "src/**/*.tsx", "../../apps/admin/src/app/contexts/confirm-dialog-context.tsx"]
```
Comment (ajay-khandla) [inline libs/react-core/tsconfig.lib.json:23]: Why need to add this ?

**Pull request merged**